libr/util/json_indent.c

```diff
@@ -3,12 +3,19 @@
 #include <r_util.h>
 #include <r_print.h>
 
-R_API char* r_print_json_indent(const char* s, bool color) {
+
+static void doIndent(int idt, char** o, const char tab) {
+	for (int i = 0; i < idt; i++) {
```

Comment:
you can't define `int` inside the for clause in C, will fail to compile in some compilers